ApiClient/Source/SchedJoulesApi.swift

```diff
@@ -64,8 +66,13 @@ public final class SchedJoulesApi: Api {
             headers[key] = value
         }
         
+        //Update the parameters to include the device uuid
+        //We first try to use the identifier for vendor to keep the uuid consistent, if we can't do it we create a random one
```

Comment:
This comment doesn't really belong here. It's more like a hind for the user of this library and should go into the documentation.